adapters/smaato/smaatotest/supplemental/bad-media-type-request.json

```diff
@@ -7,7 +7,7 @@
     "imp": [
       {
         "id": "1C86242D-9535-47D6-9576-7B1FE87F282C",
-        "native": {
+        "audio": {
```

Comment:
I don't think prebid supports `audio` as such
should we rather be bulletproof and put smth like `unsupported` here?

Reply:
changed it